qemu/tests/cfg/virtio_fs_nfs_migration_handler.cfg

```diff
@@ -0,0 +1,91 @@
+- virtio_fs_nfs_migration_handler:
+    no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1
+    no Win2008 Win7 Win2012 Win2012..r2 Win8 Win8.1
+
+    type = virtio_fs_nfs_migration_handler
+    virt_test_type = qemu
+    required_qemu = [9.0.0,)
+    kill_vm = yes
+    start_vm = yes
+    not_preprocess = yes
+
+    # local nfs config
+    setup_local_nfs = yes
+    export_options = 'rw,insecure,no_root_squash,async'
+    export_dir = /var/tmp/virtio_fs_test_nfs
+    nfs_mount_options = rw
+    nfs_mount_dir = /var/mnt/fs
+    nfs_mount_dir_targetfs = /var/mnt/targetfs
+
+    filesystems = fs
+    filesystems_migration = targetfs
+    fs_driver = virtio-fs
+    fs_source_type = mount
+    fs_source_dir_fs = ${nfs_mount_dir}
+    fs_source_dir_targetfs = ${nfs_mount_dir_targetfs}
+    fs_target = myfs
+    fs_dest = /mnt/${fs_target}
+    force_create_fs_source = no
+    remove_fs_source = no
+    fs_driver_props = {"queue-size": 1024}
+    fs_binary_extra_options = " --cache auto --migration-verify-handles --migration-confirm-paths"
+    fs_binary_extra_options_targetfs = " --cache auto"
+    driver_name = viofs
+
+    test_file = test_file
+    test_data = hello_virtiofs
+    pre_command = mkdir -p ${nfs_mount_dir_targetfs} ${nfs_mount_dir} ${export_dir}
+    pre_command += " && echo -e ${test_data} > ${export_dir}/${test_file}"
+    post_command = rm -rf ${nfs_mount_dir_targetfs} ${nfs_mount_dir} ${export_dir}
+    read_file_cmd = cat %s/${test_file}
+    cmd_tail_file = tail -f %s/${test_file}
+    tail_name = tail
+    cmd_chk_tail = ps aux | grep ${tail_name}| grep -v grep
+    cmd_kill_tail = killall -s SIGKILL tail
+
+    share_mem = yes
+    vm_mem_share = yes
+    # Note that memory-backend-file can cause error when doing live migration
+    vm_mem_backend = memory-backend-memfd
```

Comment:
@zhencliu I think the cfg still use memory-backend-memfd instead of memory-backend-file, btw I didn't find this case in our "KVM QE Feature Cases Tracker"

Reply:
Yes, as we discussed before, we have to use memory-backend-memfd instead of memory-backend-file, the latter can cause error, I didn't catch your point, you may reach out to Tingting for details on the 'KVM QE Feature Cases Tracker' issue

Reply:
thanks for the reply